src/tools/pontoon/notifications.md

```diff
@@ -8,9 +8,13 @@ When the user receives a notification, the bell icon near the user avatar in the
 
 Clicking `See all Notifications` at the bottom will take the user to the complete list of notifications. The same list can also be accessed by opening the `/notifications` URL directly (e.g. [pontoon.mozilla.org/notifications](https://pontoon.mozilla.org/notifications/)).
 
+## Receiving notifications via email
+
+Pontoon also offers the option to receive notifications via email, allowing you to stay informed about important events even when you're not actively using the platform. You can [choose to receive email notifications](users.md#user-settings) as a daily or weekly digest summary and customize your preferences by subscribing to specific notification types individually.
```

Comment:
```suggestion
Pontoon also offers the option to receive notifications via email, allowing you to stay informed about important events even when you're not actively using the platform. You can [choose to receive email notifications](users.md#user-settings) as a daily or weekly digest and customize your preferences by subscribing to specific notification types.
```

Reply:
One thing worth calling out is that this moves from "user/users can" to "you can", might be worth being consistent.